### PR TITLE
fix: remove trailing newlines from source-build results

### PR DESCRIPTION
**Pull request by tisutisu**
Addresses: https://issues.redhat.com/browse/KFLUXBUGS-1331

### CODE REVIEW

File: task/source-build/0.1/source-build.yaml [hunk 1]
```diff
@@ -96,7 +96,7 @@ spec:
           --cachi2-artifacts-dir "$CACHI2_ARTIFACTS_DIR" \
           --registry-allowlist="$registry_allowlist"
 
-        cat "$RESULT_FILE" | jq -r ".image_url" >"$RESULT_SOURCE_IMAGE_URL"
-        cat "$RESULT_FILE" | jq -r ".image_digest" >"$RESULT_SOURCE_IMAGE_DIGEST"
+        cat "$RESULT_FILE" | jq -r ".image_url" | tr -d '\n' >"$RESULT_SOURCE_IMAGE_URL"
+        cat "$RESULT_FILE" | jq -r ".image_digest" | tr -d '\n' >"$RESULT_SOURCE_IMAGE_DIGEST"
```
Comment (zregvart) [inline task/source-build/0.1/source-build.yaml:100]: Might be simpler to use `-j`:

>   -j, --join-output         implies -r and output without newline after
>                            each output;
 
```suggestion
        jq -j ".image_url" "$RESULT_FILE" >"$RESULT_SOURCE_IMAGE_URL"
        jq -j ".image_digest" "$RESULT_FILE" >"$RESULT_SOURCE_IMAGE_DIGEST"
```

Comment (tisutisu) [reply task/source-build/0.1/source-build.yaml]: Thanks. Changed it now.

**Review by tkdchen — COMMENTED** [review]
LGTM. Need to fix the ta issue by running the script.

**Comment by zregvart** [comment]
This was an issue with the `tash` tool used for generating `-oci-ta` files, should be okay now. I reran the check...

**Review by zregvart — APPROVED** [review]

**Review by mmorhun — APPROVED** [review]

**Pull request merged**